hardware_interface/src/resource_manager.cpp

```diff
@@ -1283,6 +1304,58 @@ HardwareReadWriteStatus ResourceManager::read(
   return read_write_status;
 }
 
+void ResourceManager::publish_all_interface_values() const
+{
+  control_msgs::msg::DynamicInterfaceValues interface_values;
+  interface_values.header.stamp = resource_storage_->clock_interface_->get_clock()->now();
```

Comment:
I think your messages are incomplete in the PR: https://github.com/ros-controls/control_msgs/pull/98. Please check mu suggestions : https://github.com/ros-controls/control_msgs/pull/98#discussion_r1442805951

Reply:
sry this was my fault. I changed this already, but apparently did not push the last change in the control_msgs repo